OutOfSchool/OutOfSchool.DataAccess/Repository/ChatRoomWorkshopModelForChatListRepository.cs

```diff
@@ -61,6 +61,16 @@ public Task<List<ChatRoomWorkshopForChatList>> GetByWorkshopIdAsync(Guid worksho
             return this.GetByParametersAsync(condition, searchMessagesForProvider);
         }
 
+        /// <inheritdoc/>
+        public Task<List<ChatRoomWorkshopForChatList>> GetByWorkshopIdsAsync(IEnumerable<Guid> workshopIds)
+        {
+            Expression<Func<ChatRoomWorkshop, bool>> condition = x => workshopIds.Contains(x.WorkshopId);
+
+            var searchMessagesForProvider = true;
```

Comment:
maybe move this to `GetByWorkshopIdsAsync` method arguments and have true as default value?

Reply:
Done